health/src/service.rs

```diff
@@ -14,13 +14,39 @@ use std::task::{Context, Poll, Waker};
 #[cfg(feature = "protobuf-codec")]
 use protobuf::ProtobufEnum;
 
+#[cfg(feature = "protobuf-codec")]
+use crate::proto::ServingStatus;
+
+#[cfg(feature = "protobufv3-codec")]
+use crate::proto::health_check_response::ServingStatus;
+
+#[cfg(feature = "protobuf-codec")]
+mod constants {
+    use crate::proto::ServingStatus;
+    pub const NOT_SERVING: ServingStatus = ServingStatus::NotServing;
+    pub const SERVICE_UNKNOWN: ServingStatus = ServingStatus::ServiceUnknown;
+}
+
+#[cfg(feature = "protobufv3-codec")]
+mod constants {
+    use crate::proto::health_check_response::ServingStatus;
+    pub const NOT_SERVING: ServingStatus = ServingStatus::NOT_SERVING;
+    pub const SERVICE_UNKNOWN: ServingStatus = ServingStatus::SERVICE_UNKNOWN;
```

Comment:
I think you might be able to get away with just

```suggestion
    pub use crate::proto::health_check_response::ServingStatus::*;
```

Reply:
Similar for other places where you use this trick. Alternatively, you could remove the `constants` module completely and do something like this in the branch above — I actually think that's even simpler:

```rust
    use crate::proto::ServingStatus::NotServing as NOT_SERVING;
    use crate::proto::ServingStatus::ServiceUnknown as SERVICE_UNKNOWN;
```

Reply:
Done, thanks !